app/views/admin/shared/_tabnav_product.html.haml

```diff
@@ -16,8 +16,7 @@
       = tab t("views.admin.products.tabnav.pricing_rules"), [current_facility, @product, PricePolicy], (secondary_tab == "pricing_rules")
   - if @product.is_a?(Instrument)
     = tab t("views.admin.products.tabnav.restrictions"), edit_facility_price_group_product_path(current_facility, @product), (secondary_tab == "restrictions")
-    - if @product.duration_pricing_mode?
-      = tab t("views.admin.products.tabnav.duration_pricing"), edit_facility_instrument_duration_rate_path(current_facility, @product), (secondary_tab == "duration_pricing")
+    = tab t("views.admin.products.tabnav.duration_pricing"), edit_facility_instrument_duration_rate_path(current_facility, @product), (secondary_tab == "duration_pricing")
```

Comment:
I think we should add a conditional like line 17 `- if @product.is_a?(Instrument)`

Reply:
It's indented so the one from line 17 is also applied, right?

Reply:
Right!